diagnostic.md

```diff
@@ -15,23 +16,32 @@ Place your responses inside the fenced code-blocks where indivated by comments.
   join table with references to `Movies` and `Profiles`.
 
   ```md
-    # < Your Response Here >
+  A profile has many favorites and movie is favorited many times? t
```

Comment:
Yes, we were hoping you would provide the actual table but I realize this isn't clear from the instructions